react-app/src/components/profile/ProfileButton/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch } from 'react-redux';
+import * as sessionActions from '../../../store/session';
+
+import './ProfileButton.css'
+
+function ProfileButton({ user }) {
+  const dispatch = useDispatch();
+  const [showMenu, setShowMenu] = useState(false);
+
+  const openMenu = () => {
+    if (showMenu) return;
+    setShowMenu(true);
+  };
+
+  useEffect(() => {
+    if (!showMenu) return;
+
+    const closeMenu = () => {
+      setShowMenu(false);
+    };
+
+    document.addEventListener('click', closeMenu);
+
+    return () => document.removeEventListener("click", closeMenu);
+  }, [showMenu]);
+
+  const logout = (e) => {
```

Comment:
When importing `logout` from the `session` store, this will conflict. This can be renamed to `handleLogout`